src/indexer.ts

```diff
@@ -78,8 +78,37 @@ export class Indexer {
         const error = proxy.unwrapErr();
         if (error.startsWith('[-32000] Server error: DB Not Found Error')) {
           if (blockID < currentBlockHeight) {
-            this.logger.error(error);
-            return; // a skip block, or an unavailable block on a nonarchival node
+            // Handling empty blocks
+            const emptyBlock: EmptyBlock = generateEmptyBlock(
+              blockID as number,
+              contractID.toString(),
+              this.network.chainID,
+              timestamp + 1 // arbitrary timestamp = previous block.timestamp + 1
```

Comment:
Better for skip blocks to have no timestamps (it's a nullable column) than to have arbitrary timestamps.